.github/workflows/main.yaml

```diff
@@ -462,7 +450,7 @@ jobs:
           echo "Tests run: ${{ inputs.tests_to_run }}" >> ${GITHUB_STEP_SUMMARY}
           OPERATOR_HELM_VERSION=$(helm get metadata rancher-${{ inputs.hosted_provider }}-operator -n cattle-system -o json | jq -r .version)
           echo "Installed rancher-${{ inputs.hosted_provider }}-operator chart version: $OPERATOR_HELM_VERSION" >> ${GITHUB_STEP_SUMMARY}
-          if [ ${{ inputs.tests_to_run }} == "backup_restore" ]; then
+          if [ ${{ inputs.tests_to_run }} =~ "backup_restore" ]; then
```

Comment:
not sure if this is gonna work, [ ] is posix `test` (sh) and =~ is bash

Reply:
![image](https://github.com/user-attachments/assets/1b237953-59e2-42b8-97fe-d829d0e60b17)